pkg/client/unversioned/record/event.go

```diff
@@ -222,6 +222,9 @@ func (eventBroadcaster *eventBroadcasterImpl) StartEventWatcher(eventHandler fun
 				// ever happen.
 				continue
 			}
+			if ratelimiter != nil {
+				ratelimiter.Accept()
```

Comment:
Events are often created in the middle of time-sensitive loops. I'm worried about blocking enough events here that we back up the queue and start blocking the places that create events.